src/components/Icon/AddURLIcon/AddURLIcon.tsx

```diff
@@ -2,16 +2,16 @@ import _ from 'lodash';
 import React from 'react';
 import Icon, { IIconProps, propTypes as iconPropTypes } from '../Icon';
 import { lucidClassNames } from '../../../util/style-helpers';
-import { FC, omitProps } from '../../../util/component-types';
+import { omitProps } from '../../../util/component-types';
 
 const cx = lucidClassNames.bind('&-AddURLIcon');
 
 interface IAddURLIconProps extends IIconProps {}
 
-export const AddURLIcon: FC<IAddURLIconProps> = ({
+export const AddURLIcon = ({
 	className,
 	...passThroughs
-}): React.ReactElement => {
+}: IAddURLIconProps) => {
```

Comment:
is the `React.ReactElement` return type not needed for Icons?

Reply:
There were some cases that made it weird to add an explicit return type so I called an audible and removed the eslint rule that required them. It might be over reaching now that I'm thinking about it today.

Reply:
I went ahead and restored the explicit return types to all the icons.